src/mn_setup.c

```diff
@@ -2650,35 +2674,35 @@ static setup_menu_t eq_settings1[] = {
 
     MI_GAP_EX(4),
 
-    {"Preamp dB", S_THERMO, CNTR_X, M_THRM_SPC_EQ,
+    {"Preamp dB", S_THERMO, CNTR_X, M_THRM_SPC,
      {"snd_eq_preamp"}, .action = I_OAL_EqualizerPreset},
 
     MI_GAP_EX(4),
 
-    {"Low Gain dB", S_THERMO, CNTR_X, M_THRM_SPC_EQ,
+    {"Low Gain dB", S_THERMO, CNTR_X, M_THRM_SPC,
      {"snd_eq_low_gain"}, .action = I_OAL_EqualizerPreset},
 
-    {"Mid 1 Gain dB", S_THERMO, CNTR_X, M_THRM_SPC_EQ,
+    {"Mid 1 Gain dB", S_THERMO, CNTR_X, M_THRM_SPC,
      {"snd_eq_mid1_gain"}, .action = I_OAL_EqualizerPreset},
 
-    {"Mid 2 Gain dB", S_THERMO, CNTR_X, M_THRM_SPC_EQ,
+    {"Mid 2 Gain dB", S_THERMO, CNTR_X, M_THRM_SPC,
      {"snd_eq_mid2_gain"}, .action = I_OAL_EqualizerPreset},
 
-    {"High Gain dB", S_THERMO, CNTR_X, M_THRM_SPC_EQ,
+    {"High Gain dB", S_THERMO, CNTR_X, M_THRM_SPC,
      {"snd_eq_high_gain"}, .action = I_OAL_EqualizerPreset},
 
     MI_GAP_EX(4),
 
-    {"Low Cutoff Hz", S_THERMO, CNTR_X, M_THRM_SPC_EQ,
+    {"Low Cutoff Hz", S_NUM, CNTR_X, M_SPC,
      {"snd_eq_low_cutoff"}, .action = I_OAL_EqualizerPreset},
 
-    {"Mid 1 Center Hz", S_THERMO, CNTR_X, M_THRM_SPC_EQ,
+    {"Mid 1 Center Hz", S_NUM, CNTR_X, M_SPC,
      {"snd_eq_mid1_center"}, .action = I_OAL_EqualizerPreset},
 
-    {"Mid 2 Center Hz", S_THERMO, CNTR_X, M_THRM_SPC_EQ,
+    {"Mid 2 Center Hz", S_NUM, CNTR_X, M_SPC,
      {"snd_eq_mid2_center"}, .action = I_OAL_EqualizerPreset},
 
-    {"High Cutoff Hz", S_THERMO, CNTR_X, M_THRM_SPC_EQ,
+    {"High Cutoff Hz", S_NUM, CNTR_X, M_SPC,
```

Comment:
Something I [noticed earlier](https://github.com/fabiangreffrath/woof/pull/1714#discussion_r1628511431) is that these "cutoff" frequencies are actually "corner" frequencies for shelf filters. It's apparently a mistake in the OAL documentation. I can go through the code and rename the strings and variables to be more accurate in a different PR.